test/unit/Makefile

```diff
@@ -1,9 +1,10 @@
 all:
-	bapbundle remove cwe_checker_unit_tests.plugin
-	bapbuild -r -Is analysis,checkers,utils cwe_checker_unit_tests.plugin -pkgs alcotest,yojson,unix,ppx_jane,cwe_checker_core
-	bapbundle install cwe_checker_unit_tests.plugin
-	bap ../artificial_samples/build/arrays_x64_gcc.out --pass=cwe-checker-unit-tests
-	bapbundle remove cwe_checker_unit_tests.plugin
+	bapbundle remove unit_tests_cwe_checker.plugin
+	bapbuild -r -Is analysis,checkers,utils unit_tests_cwe_checker.plugin -pkgs alcotest,yojson,unix,ppx_jane,cwe_checker_core
+	bapbundle install unit_tests_cwe_checker.plugin
+	bap tmp/arrays_gcc.out --pass=unit-tests-cwe-checker --unit-tests-cwe-checker-partial=Cconv,TypeInference,CWE476
+	bap tmp/c_constructs_gcc.out --pass=unit-tests-cwe-checker --unit-tests-cwe-checker-partial=CWE560,AddrTrans
```

Comment:
The MemRegion unit tests are missing here. They also did not run on Travis. Please re-add them somewhere.